driver-opentelemetry/src/main/java/com/datastax/driver/opentelemetry/OpenTelemetryTracingInfo.java

```diff
@@ -0,0 +1,91 @@
+package com.datastax.driver.opentelemetry;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.tracing.TracingInfo;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+
+public class OpenTelemetryTracingInfo implements TracingInfo {
+  private Span span;
+  private final Tracer tracer;
+  private final Context context;
+  private boolean tracingStarted;
+  private boolean tracingFinished;
+  private static final String mustBeInitMsg =
+      "TracingInfo.setStartTime must be called before TracingInfo.";
+
+  public OpenTelemetryTracingInfo(Tracer tracer, Context context) {
```

Comment:
Shouldn't ctor be package protected, since we have factory for that?

Reply:
Right.